docs/DeveloperGuide.md

```diff
@@ -5,6 +5,18 @@ title: Developer Guide
 * Table of Contents
 {:toc}
 
+--------------------------------------------------------------------------------------------------------------------
```

Comment:
Sorry why does this pr modify the DG?

Reply:
Will close and redo PR due to unintended updates in master branch